libsolidity/formal/SolverInterface.h

```diff
@@ -231,8 +231,11 @@ class SolverInterface
 	/// is available. Throws SMTSolverError on error.
 	virtual std::pair<CheckResult, std::vector<std::string>>
 	check(std::vector<Expression> const& _expressionsToEvaluate) = 0;
-};
 
+protected:
+	// SMT query timeout in seconds.
+	const int timeout = 1000;
```

Comment:
What is the naming convention we have for constants?

Is this really in seconds and not milliseconds? ~17 minutes seems to be quite long. I'd set it at 2 minutes, is that too short?

Reply:
Sorry, I meant milliseconds.
I was thinking of 1s per query. If it takes more than that it might take very long.

Reply:
Can we go for 10 seconds a query? Can you update the comment?

Reply:
Yep doing that now